x/registry/module_simulation.go

```diff
@@ -86,73 +86,74 @@ func (am AppModule) RegisterStoreDecoder(_ sdk.StoreDecoderRegistry) {}
 // WeightedOperations returns the all the gov module operations with their respective weights.
 func (am AppModule) WeightedOperations(simState module.SimulationState) []simtypes.WeightedOperation {
 	operations := make([]simtypes.WeightedOperation, 0)
-
-	var weightMsgCreateWhoIs int
-	simState.AppParams.GetOrGenerate(simState.Cdc, opWeightMsgCreateWhoIs, &weightMsgCreateWhoIs, nil,
-		func(_ *rand.Rand) {
-			weightMsgCreateWhoIs = defaultWeightMsgCreateWhoIs
-		},
-	)
-	operations = append(operations, simulation.NewWeightedOperation(
-		weightMsgCreateWhoIs,
-		registrysimulation.SimulateMsgCreateWhoIs(am.accountKeeper, am.bankKeeper, am.keeper),
-	))
-
-	var weightMsgBuyAlias int
-	simState.AppParams.GetOrGenerate(simState.Cdc, opWeightMsgSellAlias, &weightMsgBuyAlias, nil,
-		func(_ *rand.Rand) {
-			weightMsgBuyAlias = defaultWeightMsgSellAlias
-		},
-	)
-
-	operations = append(operations, simulation.NewWeightedOperation(
-		weightMsgBuyAlias,
-		registrysimulation.SimulateMsgBuyAlias(am.accountKeeper, am.bankKeeper, am.keeper),
-	))
-
-	var weightMsgUpdateWhoIs int
-	simState.AppParams.GetOrGenerate(simState.Cdc, opWeightMsgUpdateWhoIs, &weightMsgUpdateWhoIs, nil,
-		func(_ *rand.Rand) {
-			weightMsgUpdateWhoIs = defaultWeightMsgUpdateWhoIs
-		},
-	)
-	operations = append(operations, simulation.NewWeightedOperation(
-		weightMsgUpdateWhoIs,
-		registrysimulation.SimulateMsgUpdateWhoIs(am.accountKeeper, am.bankKeeper, am.keeper),
-	))
-
-	var weightMsgSellAlias int
-	simState.AppParams.GetOrGenerate(simState.Cdc, opWeightMsgSellAlias, &weightMsgSellAlias, nil,
-		func(_ *rand.Rand) {
-			weightMsgSellAlias = defaultWeightMsgSellAlias
-		},
-	)
-	operations = append(operations, simulation.NewWeightedOperation(
-		weightMsgSellAlias,
-		registrysimulation.SimulateMsgSellAlias(am.accountKeeper, am.bankKeeper, am.keeper),
-	))
-
-	var weightMsgTransferAlias int
-	simState.AppParams.GetOrGenerate(simState.Cdc, opWeightMsgTransferAlias, &weightMsgTransferAlias, nil,
-		func(_ *rand.Rand) {
-			weightMsgTransferAlias = defaultWeightMsgTransferAlias
-		},
-	)
-	operations = append(operations, simulation.NewWeightedOperation(
-		weightMsgTransferAlias,
-		registrysimulation.SimulateMsgTransferAlias(am.accountKeeper, am.bankKeeper, am.keeper),
-	))
-
-	var weightMsgDeactivateWhoIs int
-	simState.AppParams.GetOrGenerate(simState.Cdc, opWeightMsgDeactivateWhoIs, &weightMsgDeactivateWhoIs, nil,
-		func(_ *rand.Rand) {
-			weightMsgDeactivateWhoIs = defaultWeightMsgDeactivateWhoIs
-		},
-	)
-	operations = append(operations, simulation.NewWeightedOperation(
-		weightMsgDeactivateWhoIs,
-		registrysimulation.SimulateMsgDeactivateWhoIs(am.accountKeeper, am.bankKeeper, am.keeper),
-	))
+	/*
```

Comment:
Why is this all commented out?

Reply:
was for testing, i need to stop doing this.